### PR TITLE
fix #55284 m.imgur.com

### LINKED ISSUE #55284: m.imgur.com
### Issue URL (Annoyance)
 [https://m.imgur.com/mSkuzNf](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fm.imgur.com%2FmSkuzNf)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 2.1.2
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | Default
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Safari

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#55284

### CODE REVIEW

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -34,6 +34,7 @@ reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPa
 !
 /googleplay_135x40.png$domain=xn----etbbecbrbp5ahkja1ae7v.xn--p1ai|kingisepp-today.ru
 /appstore_135x40.png$domain=xn----etbbecbrbp5ahkja1ae7v.xn--p1ai|kingisepp-today.ru
+m.imgur.com##div[class^="SeeImgurOIA-"]
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/mobile-app.txt:37]: image

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/mobile-app.txt:37]: Scrolling works good despite of that attribute

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/mobile-app.txt:37]: Scrolling works good on ios even with this attribute.
We can add something like this
```
!+ PLATFORM(ios)
m.imgur.com##div[class^="SeeImgurOIA-"]
!+ PLATFORM (android)
m.imgur.com#?#div[class^="SeeImgurOIA-"] { display: none!important; }
m.imgur.com#?#body {overflow-y: hidden!important; }
```

Comment (Alex-302) [reply AnnoyancesFilter/sections/mobile-app.txt:37]: > We can add something like this

ok

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 2]
```diff
@@ -630,6 +630,11 @@ shiftdelete.net##img[title="Torrent Yasal mı? - ShiftDelete.Net"]
 mediamarkt.de##.mrh-app-banner
 rt.com###offers
 !
+!+ NOT_PLATFORM (ext_android_cb)
+m.imgur.com##div[class^="SeeImgurOIA-"]
+!+ PLATFORM (android)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/mobile-app.txt:635]: no need in this hint. AG extension in Mobile FF support this rule.

Comment (Alex-302) [reply AnnoyancesFilter/sections/mobile-app.txt:635]: And do not add spaces between `PLATFORM` and `(` We do not do it in the rules.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 3]
```diff
@@ -630,6 +630,10 @@ shiftdelete.net##img[title="Torrent Yasal mı? - ShiftDelete.Net"]
 mediamarkt.de##.mrh-app-banner
 rt.com###offers
 !
+!+ NOT_PLATFORM(ext_android_cb)
+m.imgur.com##div[class^="SeeImgurOIA-"]
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/mobile-app.txt:634]: just make CSS rule

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**